fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordContext.java

```diff
@@ -1388,4 +1392,14 @@ public synchronized void run() {
         return (T) session.remove(key);
     }
 
+    /**
+     * Get the properties configured by adopter for this FDBRecordContext.
+     *
+     * @return the wrapper of a mapping of the properties
+     */
+    @API(API.Status.EXPERIMENTAL)
+    public RecordLayerPropertyStorage getPropertyStorage() {
```

Comment:
No need to change this, but just a personal opinion:  It seems strange and not very useful to prefix classes with the name of the project, as we already have package namespacing to differentiate it from any other class that may also be called `PropertyStorage`.